packages/wdio-cucumber-framework/src/types.ts

```diff
@@ -133,8 +133,16 @@ export interface HookParams {
     step?: PickleStep | TestStep
 }
 
+export interface HookOptions {
+    name?: string,
+    tags?: string[],
+    hookType?: string,
+    body: string
+}
+
 export interface StepDefinitionOptions {
-    retry: number
+    retry: number,
+    _wdioOptions?: HookOptions
```

Comment:
What is the parameter suppose to do? Is it just internally used or also for users?